src/plays/pokemon-stats/components/Search.tsx

```diff
@@ -1,31 +1,37 @@
 import React from "react";
 import "./search.css";
-const search = (props) => {
-  const { setname, fetchPokemonDetails, name } = props;
+type SearchProps = {
+  eventchange:(e: React.ChangeEvent<HTMLInputElement>) => void;
+  fetchPokemonDetails: (e: React.FormEvent<HTMLFormElement>) => void;
+  Name: string;
+};
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {}
+
+const search = (props: SearchProps) => {
```

Comment:
@yung-coder , the Component name should be capitalized.
```suggestion
const Search = (props: SearchProps) => {
```